### PR TITLE
*added support for offchain indexing to node helm chart

### PR DESCRIPTION
**Pull request by okalenyk**

### CODE REVIEW

**Comment by PierreBesson** [comment]
Thanks for the contribution @okalenyk. As a standard feature of substrate, this should be added to the helm chart.

File: charts/node/values.yaml [hunk 1]
```diff
@@ -449,6 +449,10 @@ node:
     #   mountPath: /etc/config/
     #   readOnly: true
 
+  # Enable Offchain Indexing.
+  # https://docs.substrate.io/fundamentals/offchain-operations/
+  offchainIndexing: false
```
Comment (PierreBesson) [inline charts/node/values.yaml:454]: Can you please rename this to `enableOffchainIndexing`  to be consistent with the flag.

**Review by PierreBesson — CHANGES_REQUESTED** [review]
Can you also add the `--enable-offchain-indexing` flag to the regex in https://github.com/paritytech/helm-charts/blob/main/charts/node/templates/_helpers.tpl#L103
This will validate that the flag is not already passed via `node.flags`.

File: charts/node/values.yaml (hunk 1 above)
Comment (okalenyk) [reply charts/node/values.yaml]: Done. Thanks for reviewing.

**Comment by PierreBesson** [comment]
Thank you can you also bump the minor version of the chart for the auto-release ?

**Comment by okalenyk** [comment]
> Thank you! Can you also bump the minor version of the chart for the auto-release ?

Sure. Bumped to 4.4.1.

**Review by PierreBesson — APPROVED** [review]

**Pull request merged**